src/Settings/AdvancedParameters.as

```diff
@@ -24,9 +24,30 @@ namespace PluginSettings
         RMC_MX_Url = UI::InputText("MX Base URL", RMC_MX_Url);
         UI::SetPreviousTooltip("Use this URL for API calls to ManiaExchange. Useful for hosting your own service for caching and preloading API responses for better performance.\nOnly change if you know what you're doing!");
 
+#if TMNEXT
+        if (UI::Button("Use official TMX API")) {
+            RMC_MX_Url = "https://" + MX_URL;
+        }
+
+        // Commented for the moment since it's not ready yet
```

Comment:
I'll prefer wait until it's ready to start merging this PR :)

Reply:
Sounds good